sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -145,23 +154,52 @@ struct protocol_key {
       protocol_key, std::integral_constant<protocol_name, Protocol>>;
 };
 
-template <int Latency>
+template <unsigned Latency>
 inline constexpr ready_latency_key::value_t<Latency> ready_latency;
 
-template <int Bits>
+template <unsigned Bits>
 inline constexpr bits_per_symbol_key::value_t<Bits> bits_per_symbol;
 
 template <bool Valid>
 inline constexpr uses_valid_key::value_t<Valid> uses_valid;
 
+template <bool Ready>
+inline constexpr uses_ready_key::value_t<Ready> uses_ready;
+
 template <bool HighOrder>
 inline constexpr first_symbol_in_high_order_bits_key::value_t<HighOrder>
     first_symbol_in_high_order_bits;
 
 template <protocol_name Protocol>
 inline constexpr protocol_key::value_t<Protocol> protocol;
 
-} // namespace sycl::ext::intel::experimental
+namespace avalon-st {
+  using sycl::ext::intel::experimental::bits_per_symbol;
+  using sycl::ext::intel::experimental::bits_per_symbol_key;
```

Comment:
I don't think we need the keys in this namespace as we don't expect users to use them directly.